src/PhpDoc/ResolvedPhpDocBlock.php

```diff
@@ -297,6 +302,7 @@ public function changeParameterNamesByMapping(array $parameterNameMapping): self
 		$self->typeAliasTags = $this->typeAliasTags;
 		$self->typeAliasImportTags = $this->typeAliasImportTags;
 		$self->assertTags = $assertTags;
+		$self->selfOutTypeTag = $this->getThisOutTag();
```

Comment:
All the other properties here assign properties, so it should be consistent. That's because we don't need to resolve the tag at this point, let `false` be `false`.